src/main/java/ludum/dare/components/PhysicsComponent.java

```diff
@@ -25,5 +34,8 @@ public void update(float delta) {
         pos.x = body.aabb.xy.x;
         pos.y = body.aabb.xy.y;
         size.set(body.aabb.width, body.aabb.height);
+
+        // yes... I know, its ugly... but screw it, it works
+        if (animationComponent != null) animationComponent.setFlipVerticalAxis(body.facing == Facing.LEFT);
```

Comment:
:( Can we move this check to the animation component instead? Makes more sense there.